From 984dd689b012554683bdae9fe5cfb51120c2c48e Mon Sep 17 00:00:00 2001
From: Sebastian Bauer <mail@sebastianbauer.info>
Date: Sun, 3 Jan 2016 10:03:39 +0100
Subject: [PATCH 9/9] Removed fsync().

It is not needed for current version of newlib.library.
---
 lib/amigaos.c | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/lib/amigaos.c b/lib/amigaos.c
index 37f5ac19ca1aa5ea3fd1d9a4dd666bfb2de6d710..8c9a6a257302e579d8356d08132eb52f7a28adb5 100644
--- a/lib/amigaos.c
+++ b/lib/amigaos.c
@@ -415,17 +415,12 @@ int symlink(const char *oldpath, const char *newpath)
 	if (DOSFALSE == IDOS->MakeLink((STRPTR)newpath, (STRPTR)oldpath, LINK_SOFT))
 		return -1;
 
 	return 0;
 }
 
-int fsync(int fd)
-{
-	fflush(fd);
-}
-
 int statfs (const char *path, struct statfs *buf)
 {
 	char buffer[1000];
 	BPTR lock;
 	struct InfoData ifd;
 	char *type;
-- 
2.1.4

